packages/cli/src/cli.ts

```diff
@@ -115,6 +116,23 @@ program
         await checkIntegrationBuild();
     });
 
+const openAPIProgram = program.command('openapi').description('manage OpenAPI specifications');
```

Comment:
```suggestion
const openAPIProgram = program.command('openapi').description('Manage OpenAPI specifications');
```

Reply:
There is no uppercase anywhere, that's why I kept the same style.

Reply:
If we do, we should do it for everything, else it's weird.